server/app.js

```diff
@@ -1,48 +1,54 @@
-const createError = require('http-errors');
-const express = require('express');
-const path = require('path');
-const cookieParser = require('cookie-parser');
-const logger = require('morgan');
-const serveStatic = require("serve-static");
-const indexRouter = require('./routes/index');
-const usersRouter = require('./routes/users');
+var createError = require("http-errors");
+var express = require("express");
+var path = require("path");
+var cookieParser = require("cookie-parser");
+var logger = require("morgan");
+var cors = require("cors");
 
-const app = express();
+var indexRouter = require("./routes/index");
+var usersRouter = require("./routes/users");
+var mapRouter = require("./routes/map");
+var app = express();
+
+const rateLimit = require("express-rate-limit");
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'ejs');
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "jade");
 
-app.use(logger('dev'));
+app.use(cors());
+app.use(logger("dev"));
 app.use(express.json());
-app.use(express.urlencoded({extended: false}));
+app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(serveStatic(path.join(__dirname, '../client/build')));
-
-app.get('*', (req, res, next) => {
-  console.log(path.join(__dirname, "../client/build", "index.html"));
-  res.sendFile(path.join(__dirname, "../client/build", "index.html"));
-});
+app.use(express.static(path.join(__dirname, "public")));
 
-//app.use(express.static(path.join(__dirname, 'public')));
-app.use('/', indexRouter);
-app.use('/users', usersRouter);
+app.use("/", indexRouter);
+app.use("/users", usersRouter);
+app.use("/map", mapRouter);
 
 // catch 404 and forward to error handler
-app.use(function (req, res, next) {
-  next(createError(404));
+app.use(function(req, res, next) {
+    next(createError(404));
 });
 
 // error handler
-app.use(function (err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get('env') === 'development' ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.render('error');
+app.use(function(err, req, res, next) {
+    // set locals, only providing error in development
+    res.locals.message = err.message;
+    res.locals.error = req.app.get("env") === "development" ? err : {};
+
+    // render the error page
+    res.status(err.status || 500);
+    res.render("error");
+});
+
+const limiter = rateLimit({
```

Comment:
I think the comments aren't accurate here, right?  It looks like this is saying 5 requests in 15 minute period, right?